Metadata.yaml

```diff
@@ -82,23 +82,31 @@ Authors:
   Core:
     Description: |
       For the time a Developer serves as Core developer they take on some
-      project-management responsibilities, such as merging pull-requests.
+      project-management responsibilities, such as initiating code reviews,
+      merging pull-requests, maintaining compatibility with third-party
```

Comment:
I would like you to remove "merging pull requests"

Reply:
reworded the fixup

Reply:
Looks good, thanks!